javascript/js-inputs-and-strings_switch-values/js/index.js

```diff
@@ -14,13 +14,27 @@ Hint: the HTML entity for the switch button is: &updownarrow
 */
 
 const firstInput = document.querySelector('[data-js="first-input"]');
+const secondInput = document.querySelector('[data-js="second-input"]');
 const uppercaseButton = document.querySelector('[data-js="button-uppercase"]');
 const lowercaseButton = document.querySelector('[data-js="button-lowercase"]');
+const checkButton = document.querySelector('[data-js="button-check"]');
 
 uppercaseButton.addEventListener("click", () => {
   firstInput.value = firstInput.value.toUpperCase();
+  secondInput.value = secondInput.value.toUpperCase();
 });
 
 lowercaseButton.addEventListener("click", () => {
   firstInput.value = firstInput.value.toLowerCase();
+  secondInput.value = secondInput.value.toLowerCase();
+});
+
+checkButton.addEventListener("click", (e) => {
+  //firstInput.value = secondInput.value;
+  e.preventDefault();
```

Comment:
You don't need to handle the event. because the type of the button is `button`
This only applies if the type is `submit` or is not defined